src/components/Letter.tsx

```diff
@@ -387,6 +372,18 @@ export function LetterView({
           </div>
         </div>
         <div className="stamps">
+          {isEditable && <select
+            className="letterTypeSelect"
+            value={letterType}
+            onChange={(e) => setType(e.target.value as LetterType)}
+          >
+            {Object.values(LetterType).map((type) => (
+              <option key={type} value={type}>
+                {LetterTypeToDisplay[type]}
+              </option>
+            ))}
+          </select>
+          }
```

Comment:
too extreme padding here on te memo style only two lines show and it doesnt expand in height
<img width="442" alt="image" src="https://user-images.githubusercontent.com/14796580/219903116-a05d1163-2078-457e-99bf-44e0bbdefff6.png">